sdk/include/psdk/CMakeLists.txt

```diff
@@ -88,6 +91,7 @@ list(APPEND SOURCE
     oaidl.idl
     objectarray.idl
     objidl.idl
+    objidlbase.idl
```

Comment:
Is this file already present in our codebase (even if it wasn't included there before)?